File: LuminaBeauty/LuminaBeauty-Model/src/main/java/pe/edu/pucp/luminaBeauty/Model/Usuario.java
package pe.edu.pucp.luminaBeauty.Model;

public class Usuario {
    private int id;
    private String nombre;
    private String apellido;
    private String correo;
    private String contrasena;
    private String dni;
    private String telefono;
    private int estado;

    public Usuario() {
    }

    // ID
    public int getId() { return id; }
    public void setId(int id) { this.id = id; }

    // Nombre
    public String getNombre() { return nombre; }
    public void setNombre(String nombre) { this.nombre = nombre; }

    // Apellido
    public String getApellido() { return apellido; }
    public void setApellido(String apellido) { this.apellido = apellido; }

    // Correo
    public String getCorreo() { return correo; }
    public void setCorreo(String correo) { this.correo = correo; }

    // Contraseña
    public String getContrasena() { return contrasena; }
    public void setContrasena(String contrasena) { this.contrasena = contrasena; }

    // DNI
    public String getDni() { return dni; }
    public void setDni(String dni) { this.dni = dni; }

    // Teléfono
    public String getTelefono() { return telefono; }
    public void setTelefono(String telefono) { this.telefono = telefono; }

    // Estado
    public int getEstado() { return estado; }
    public void setEstado(int estado) { this.estado = estado; }
}

File: lumina_beauty_final_version/lumina_beauty_frontend/LuminaBeauty/Servicios/Modelo/Usuario.cs
using System.Text.Json.Serialization;

namespace LuminaBeauty.Servicios.Modelo
{
    public class Usuario
    {
        [JsonPropertyName("id_usuario")]
        public int Id { get; set; }

        [JsonPropertyName("nombres")]
        public string Nombre { get; set; } = string.Empty;

        [JsonPropertyName("apellidos")]
        public string Apellido { get; set; } = string.Empty;

        [JsonPropertyName("correo")]
        public string Correo { get; set; } = string.Empty;

        [JsonPropertyName("contrasena_hash")]
        public string Contrasena { get; set; } = string.Empty;

        [JsonPropertyName("telefono")]
        public string Telefono { get; set; } = string.Empty;

        [JsonPropertyName("dni")]
        public string Dni { get; set; } = string.Empty;

        [JsonPropertyName("tipo_usuario")]
        public string TipoUsuario { get; set; } = string.Empty;

        [JsonPropertyName("estado")]
        public int Estado { get; set; } = 1;
    }
}
